app/uk/gov/hmrc/minorentityidentificationfrontend/connectors/CreateBusinessVerificationJourneyConnector.scala

```diff
@@ -57,35 +60,41 @@ class CreateBusinessVerificationJourneyConnector @Inject()(http: HttpClient,
     )
   }
 
+  private def jsonPartsBy(businessEntity: BusinessEntity, journeyId: String): (String, Call) = businessEntity match {
```

Comment:
Personally, I wouldn't evaluate the identifier key and continueUrl at this point. I would define the continue url in the CheckYourAnswers controller and pass it down to the submission service. Then I'd define the key identifier in the submission service and pass both down to the connector. The reason for this is that at present the connector is using the controller routes information. Adhering to the principle of separation of concerns routing stuff should be evaluated at the controller level. As such I'd rather just pass down the continue url from the controller.

Reply:
Richard I think that they are just a couple of urls ... the fact that to "retrieve" them there is a global value that has the name controller in it, it does not make any difference.
Separation of concern if we were doing something related to Http Request, Http Response ... Http status for sure it would have make more the sense, not in this situation.
If you want me to hide the concept of continue url I can it behind the AppConfig (to do a dirty thing) or a trait to do I clean thing.
Richard let's decide tomorrow with the team after stand up to see what they thing.

Reply:
The team didn't seem too concerned.